### ISSUE TITLE
aria-roledescription is flagged as invalid

### ISSUE THREAD

**Issue by CamiloGarciaLaRotta** [GoogleChrome/lighthouse#9113]
For the following website: https://camilogarcialarotta.github.io/
The message is: 
```
`[aria-*]` attributes are not valid or misspelled
```

`aria-roledescription` is a valid tag as per: https://www.w3.org/TR/wai-aria-1.1/#aria-roledescription

**Comment by robdodson** [comment]
Moved this over to the Lighthouse repo. Lighthouse was crashing for my in Canary so I wasn't able to test again :\

**Comment by patrickhulce** [comment]
Latest Lighthouse (axe-core 3.2.2), fails on "[aria-*] attributes do not match their roles" (`aria-allowed-attr`).

![image](https://user-images.githubusercontent.com/2301202/58842363-6f48b200-8633-11e9-83c1-c922d14f2e37.png)

This is also reproducible in the axe extension, so we'll need a fix in axe-core for this.
![image](https://user-images.githubusercontent.com/2301202/58842451-e2eabf00-8633-11e9-9dc7-43267a5c561e.png)

**Comment by patrickhulce** [comment]
@straker did you already file any issue for this we can reference or do we need to file one?

**Comment by pepinho24** [comment]
Hello, 

Is it possible that this is related to the fact that a `role="combobox"`  with `aria-readonly` attribute fails with the `[aria-*] attributes do not match their roles` when running the Lighthouse audit?

It used to be not allowed in [ARIA 1.0](https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-readonly), but is allowed in [ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#aria-readonly). 

I have checked the source code of the AXE you are using and it seems it is not added: https://github.com/dequelabs/axe-core/blob/a4b524084cb8fd61e5ac802b3b6bc7a0d5f86668/lib/commons/aria/index.js#L436

The same is observed with role="listbox" also.

**Comment by straker** [comment]
@patrickhulce yep. https://github.com/dequelabs/axe-core/issues/1517

**Comment by connorjclark** [comment]
Issue has been resolved.

**Comment by Abdull** [comment]
> Issue has been resolved.

In which Lighthouse version is _this_ issue supposed to be resolved?

When running Lighthouse 5.7.0 with Chrome 81, I still get `[aria-*] attributes do not match their roles` for the following websites:

* https://camilogarcialarotta.github.io/
* https://www.w3.org/TR/wai-aria-practices-1.1/examples/carousel/carousel-1.html

**Comment by patrickhulce** [comment]
@Abdull fixed in 6.0.0-beta.0 not yet released to Chrome.

**Comment by godanny86** [comment]
Ah just found: https://web.dev/lighthouse-whats-new-6.0/, so pretty soon this fix will be in Chrome Stable?
